tests/test_service.py

```diff
@@ -26,12 +26,19 @@ def test_check_convert_to_string(self):
         expect(lambda: isinstance(_convert_string("Hello world"), str))
         assert_expectations()
 
-    def test_list_to_payload(self):
-        """Test convert dict to list of dicts."""
-        initial_dict = {'key': "value", 'key1': 'value1'}
-        expected_list = [{'key': 'key', 'value': 'value'},
-                         {'key': 'key1', 'value': 'value1'}]
-        assert _list_to_payload(initial_dict) == expected_list
+    def test_dict_to_payload_without_system_key(self):
```

Comment:
These two tests can be combined into the one function wrapped with the parametrize marker.

```
    @pytest.mark.parametrize('system', [True, False])
    def test_dict_to_payload_with_system_key(self, system):
        """Test convert dict to list of dicts with key system."""
        initial_dict = {"aa": 1, "b": 2, "system": system}
        expected_list = [{'key': 'aa', 'value': '1', 'system': system},
                         {'key': 'b', 'value': '2', 'system': system}]
        assert _dict_to_payload(initial_dict) == expected_list
```